charts/playbooks-kubernetes/templates/cordon-node.yaml

```diff
@@ -0,0 +1,46 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flanksource/duty/main/schema/openapi/playbook.schema.json
+
+{{- if and .Values.playbooks.enabled .Values.playbooks.cordonNode }}
+---
+apiVersion: mission-control.flanksource.com/v1
+kind: Playbook
+metadata:
+  name: kubernetes-cordon-node
+spec:
+  title: Cordon Node
+  icon: k8s-node
+  category: Kubernetes
+  configs:
+    - agent: all
+      types:
+        - Kubernetes::Node
+
+  parameters:
+    - name: selector
```

Comment:
I don't think select makes sense here, because we have already selected the node config item
Dry run on a cordon is probably also overkill - For a drain yeh it makes sense